scene/gui/scroll_container.cpp

```diff
@@ -281,6 +281,7 @@ void ScrollContainer::_gui_focus_changed(Control *p_control) {
 	if (follow_focus && is_ancestor_of(p_control)) {
 		ensure_control_visible(p_control);
 	}
+	queue_redraw();
```

Comment:
This will cause redraw of *every* ScrollContainer every time focus changes. You could remember whether the container was focused, to only redraw when the focus changes. Also if focus border is disabled, no redrawing should happen.